### ISSUE TITLE
No functions in HACS

### ISSUE THREAD

**Issue by Micha-DU** [hacs/integration#3270]
### Web browser

Firefox, Chrome, Microsoft Edge

### Web browser version

117.0.1

### System Health details

I´m sorry, i don´t have "system health" in my HA installation :-(

### Checklist

- [X] I'm running the newest version of HACS <https://github.com/hacs/integration/releases/latest>
- [X] I have filled out the issue template to the best of my ability.
- [X] I have read <https://hacs.xyz/docs/issues>
- [X] This issue is related to the frontend of HACS.
- [X] This issue only contains 1 issue (if you have multiple issues, open one issue for each issue).
- [X] This issue is not a duplicate issue of currently [open](https://github.com/hacs/integration/issues) or issues [pending release](https://github.com/hacs/integration/issues?q=is%3Aissue+is%3Aclosed+sort%3Aupdated-desc+milestone%3Anext).

### Describe the issue

I don´t have any function in my HACS like "integrations" or so. Choosing HACS in the left side bar, then appears only one column right like shown on the screenshot below.

### Reproduction steps

no reproduction, it´s always

### Screenshots

![Screenshot 2023-09-25 073021](https://github.com/hacs/integration/assets/118131824/cf5e046f-97cc-4276-a24a-8eccb4fa1cc4)

### Javascript logs from your browser console

```text
?
```

**Comment by ludeeus** [comment]
Thats expected when you enable experimental.

**Comment by Micha-DU** [comment]
I didn´t activate "experimental"...!!!

**Comment by ludeeus** [comment]
You did, or you would not see that.

**Comment by Micha-DU** [comment]
OMG i´m very sorry...! You are right... i re-installed after a "clear" remove and didn´t mark the last checkbox ("experimental...")... now it´s runnig ;-)

Thank you so much!!!!

**Comment by ludeeus** [comment]
It was running before as well, it just looks different.

**Comment by Acr515** [comment]
I'm a bit frustrated because I enabled experimental when I installed HACS for the first time, but I was having trouble using the UI and wanted to roll back. I unchecked the experimental checkbox, flushed my cache, rebooted HA, but the UI did not revert back. There seems to be no documentation at all about how to use components based on the new HACS UI, and as somebody new to all of this, I'm unsure of what I'm supposed to be doing.
